models/notification.go

```diff
@@ -89,8 +89,14 @@ func (opts *FindNotificationOptions) ToCond() builder.Cond {
 	if opts.IssueID != 0 {
 		cond = cond.And(builder.Eq{"notification.issue_id": opts.IssueID})
 	}
-	if opts.Status != 0 {
-		cond = cond.And(builder.Eq{"notification.status": opts.Status})
+	if len(opts.Status) == 1 {
```

Comment:
cant we just use an "IN" condition if Status != 0 ?

Reply:
Yes, but there are only 3 status types in total.

Reply:
yes so it is fine for sqlite too - I dont see why we add go complexytiy what can be reached with a simple SQL statement - but thats only my point of view :)

Reply:
done